src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -24,14 +24,15 @@ static char       CCSN_Prof_File[MAX_STRING];      // filename of input profile
 static double    *CCSN_Prof = NULL;                // radial profile of initial condition
 static int        CCSN_Prof_NBin;                  // number of radial bins in the input profile
 static int        CCSN_NCol;                       // number of columns read from the input profile
-static int       *CCSN_TargetCols = new int [7];   // index of columns read from the input profile
-static int        CCSN_ColIdx_R;                   // column index of radius          in the input profile
-static int        CCSN_ColIdx_Dens;                // column index of density         in the input profile
-static int        CCSN_ColIdx_Pres;                // column index of pressure        in the input profile
-static int        CCSN_ColIdx_Velr;                // column index of radial velocity in the input profile
-static int        CCSN_ColIdx_Ye;                  // column index of Ye              in the input profile
-static int        CCSN_ColIdx_Temp;                // column index of temperature     in the input profile
-static int        CCSN_ColIdx_Entr;                // column index of entropy         in the input profile
+static int       *CCSN_TargetCols = new int [8];   // index of columns read from the input profile
+static int        CCSN_ColIdx_R;                   // column index of radius           in the input profile
+static int        CCSN_ColIdx_Dens;                // column index of density          in the input profile
+static int        CCSN_ColIdx_Pres;                // column index of pressure         in the input profile
+static int        CCSN_ColIdx_Velr;                // column index of radial velocity  in the input profile
+static int        CCSN_ColIdx_Ye;                  // column index of Ye               in the input profile
+static int        CCSN_ColIdx_Temp;                // column index of temperature      in the input profile
+static int        CCSN_ColIdx_Entr;                // column index of entropy          in the input profile
+static int        CCSN_ColIdx_Omega;               // column index of rotational velocity in the input profile
```

Comment:
- [ ] Replace "rotational velocity" with "angular frequency"
- [ ] Align with the above lines

Reply:
I updated the comments.